### PR TITLE
[Feature][AES-1511] Create Product Extract Component for How to Use section [Add Flex for image]

### PR DESCRIPTION
**Pull request by hassanraza0054**
Included Flex logic to make image with dynamic while setting max height to 50px as per requirement.

### CODE REVIEW

**Review by zack76 — CHANGES_REQUESTED** [review]
Screen Shot 2020-09-17 at 4 53 54 pm
One border is missing, could you double-check with it?

Also the width of this slice looks more than 100%

**Comment by hassanraza0054** [comment]
> Screen Shot 2020-09-17 at 4 53 54 pm
> 
> One border is missing, could you double-check with it?
> Also the width of this slice looks more than 100%

Ah the DefinitionListWithProductExtract file is just a showcase component. If you want to see the borders, just tick the relevant boxes in the knobs! 

Let me have a look at the width issue.

**Comment by hassanraza0054** [comment]
Just adding a note that the width seems to be alright from my end. Made another commit to correct fixtures and add image properties as per your suggestion. Thanks @zack76

File: src/components/ProductExtract/__snapshots__/ProductExtract.spec.js.snap [hunk 1]
```diff
@@ -11,36 +11,50 @@ exports[`<ProductExtract /> renders base component correctly 1`] = `
...
-    <h4
-      className="base xXSmall dark heading delayAnimation slideIn"
-    >
-      <span>
-        Works well with
-      </span>
-    </h4>
     <div
-      className="productName delayAnimation slideIn"
+      className="test delayAnimation slideIn"
```
Comment (kevin-ho87) [inline src/components/ProductExtract/__snapshots__/ProductExtract.spec.js.snap:15]: Where is this `test` coming from?

File: src/components/ProductExtract/ProductExtract.js [hunk 2]
```diff
@@ -32,6 +31,11 @@ const ProductExtract = forwardRef(function ProductExtractRef(
 
   const linkClassSet = cx(styles.link);
 
+  const flexClassSet = cx(styles.test, styles.delayAnimation, {
```
Comment (zack76) [inline src/components/ProductExtract/ProductExtract.js:34]: `test` here? @kevin-ho87 
should be deleted i think @hassanraza0054

**Review by zack76 — CHANGES_REQUESTED** [review]

File: src/components/ProductExtract/__snapshots__/ProductExtract.spec.js.snap (hunk 1 above)
Comment (beau-charman) [reply src/components/ProductExtract/__snapshots__/ProductExtract.spec.js.snap:15]:
```
.test {
  display: flex;
  justify-content: space-between;
}
```

Comment (hassanraza0054) [reply src/components/ProductExtract/__snapshots__/ProductExtract.spec.js.snap]: Change made! Sorry for that and thanks for the catch!

File: src/components/ProductExtract/ProductExtract.js (hunk 2 above)
Comment (hassanraza0054) [reply src/components/ProductExtract/ProductExtract.js]: Done! Thanks for the catch

**Review by zack76 — APPROVED** [review]

**Review by kevin-ho87 — APPROVED** [review]

**Pull request merged**